Program.cs

```diff
@@ -105,6 +110,7 @@ public static void RemovePerson()
             }
         }
 
+        // person book program
```

Comment:
PersonBookExe shouldn't be a method's name. you should find better name for it!

Reply:
changed it RunProgram